managed/devops/bin/yb_platform_util.sh

```diff
@@ -0,0 +1,525 @@
+then
+    case $1 in
+        -h|--help)
+            help="True"
+            shift # argument
+            ;;
+        *)
+            operation=$(echo "$1" | tr '[:lower:]' '[:upper:]')
+            shift # argument
+            ;;
+    esac
+else
+    echo "Error: Invalid Actions."
+    HelpMessage
+    exit
+fi
+# unset customer_uuid
+# unset universe_uuid
+# unset universe_name
+# unset input_file
+# unset help
+# unset confirmation
+# unset skip_wait
+# unset task_id
+
+# Loop to fetch all the values fro the user
+# Example, sh yb_platform_util.sh -c f33e3c9b-75ab-4c30-80ad-cba85646ea39 -n jd-script-8-2-21 -o GET
+# this will set customer id name and opertation as get.
+while [[ "$#" -gt 0 ]]; do
+    case $1 in
+        -c|--customer_uuid)
+            customer_uuid="$2"
+            shift # argument
+            shift # value
+            ;;
+        -u|--universe_uuid)
+            universe_uuid="$2"
+            shift # argument
+            shift # value
+            ;;
+        -n|--universe_name)
+            universe_name="$2"
+            shift # argument
+            shift # value
+            ;;
+        -f|--file)
+            input_file="$USER_DIR/$2"
+            shift # argument
+            shift # value
+            ;;
+        -h|--help)
+            help="True"
+            shift # argument
+            ;;
+        -y|--yes)
+            confirmation="y"
+            shift # argument
+            ;;
+        --no-wait)
+            skip_wait="y"
+            shift # argument
+            ;;
+        -t|--task)
+            task_id="$2"
+            shift # argument
+            shift # value
+            ;;
+        *)
+            echo "Error: Invalid command line argument"
+            HelpMessage
+            exit 255
+            ;;
+    esac
+done
+
+
+# Checking if system has python3 installed.
+# If python3 installed run script using python 3 use 2 otherwise.
+
+python_3="False"
+if [[ "$(python3 -V)" =~ "Python 3" ]]
+then
+    python_3="True"
+fi
+
+# Help message for the users.
+if [[ ! -z "$help" ]] && [[ $help == "True" ]]
+then
+    HelpMessage
+    exit 255
+fi
+
+# Validation to set YB_PLATFORM_URL and YB_PLATFORM_API_TOKEN UUID
+if [[ -z "$YB_PLATFORM_URL" ]]
+then
+    printf "YB_PLATFORM_URL is not set. Set YB_PLATFORM_URL as env variable to proceed.\n"
+    printf "\n\texport YB_PLATFORM_URL=<base_url_of_platform>\n\nExample:"
+    printf "\n\texport YB_PLATFORM_URL=http://localhost:9000\n"
+    exit 255
+fi
+
+if [[ -z "$YB_PLATFORM_API_TOKEN" ]]
+then
+    printf "YB_PLATFORM_API_TOKEN is not set. Set YB_PLATFORM_API_TOKEN as env variable to proceed.\n"
+    printf "\n\texport YB_PLATFORM_API_TOKEN=<platform_api_token>\n\nExample:"
+    printf "\n\t export YB_PLATFORM_API_TOKEN=e16d75cf-79af-4c57-8659-2f8c34223551\n"
+    exit 255
+fi
+
+base_url=$(echo "$YB_PLATFORM_URL" | sed 's/\/$//')
+auth_uuid="$YB_PLATFORM_API_TOKEN"
+
+if [[ -z "$customer_uuid" ]]
+then
+    if [[ $python_3 == "True" ]]
```

Comment:
See one of my previous comments - if we use two variables instead of one `python_3` then it will be simpler.